modules/flowable-bpmn-converter/src/test/java/org/flowable/editor/language/xml/SubProcessConverterAutoLayoutTest.java

```diff
@@ -54,32 +51,31 @@ protected String getResource() {
 
     private void validateModel(BpmnModel model) {
         FlowElement flowElement = model.getMainProcess().getFlowElement("start1");
-        assertNotNull(flowElement);
-        assertTrue(flowElement instanceof StartEvent);
-        assertEquals("start1", flowElement.getId());
+        assertThat(flowElement)
+                .isInstanceOfSatisfying(StartEvent.class, startEvent -> {
+                    assertThat(startEvent.getId()).isEqualTo("start1");
+                });
 
         flowElement = model.getMainProcess().getFlowElement("userTask1");
-        assertNotNull(flowElement);
-        assertTrue(flowElement instanceof UserTask);
-        assertEquals("userTask1", flowElement.getId());
-        UserTask userTask = (UserTask) flowElement;
-        assertEquals(1, userTask.getCandidateUsers().size());
-        assertEquals(1, userTask.getCandidateGroups().size());
+        assertThat(flowElement)
+                .isInstanceOfSatisfying(UserTask.class, userTask -> {
+                    assertThat(userTask.getId()).isEqualTo("userTask1");
+                    assertThat(userTask.getCandidateUsers()).hasSize(1);
+                    assertThat(userTask.getCandidateGroups()).hasSize(1);
+                });
 
         flowElement = model.getMainProcess().getFlowElement("subprocess1");
-        assertNotNull(flowElement);
-        assertTrue(flowElement instanceof SubProcess);
-        assertEquals("subprocess1", flowElement.getId());
-        SubProcess subProcess = (SubProcess) flowElement;
-        assertEquals(6, subProcess.getFlowElements().size());
-
-        List<ValuedDataObject> dataObjects = ((SubProcess) flowElement).getDataObjects();
-        assertEquals(1, dataObjects.size());
-
-        ValuedDataObject dataObj = dataObjects.get(0);
-        assertEquals("SubTest", dataObj.getName());
-        assertEquals("xsd:string", dataObj.getItemSubjectRef().getStructureRef());
-        assertTrue(dataObj.getValue() instanceof String);
-        assertEquals("Testing", dataObj.getValue());
+        assertThat(flowElement)
+                .isInstanceOfSatisfying(SubProcess.class, subProcess -> {
+                    assertThat(subProcess.getId()).isEqualTo("subprocess1");
+                    assertThat(subProcess.getFlowElements()).hasSize(6);
+                    assertThat(subProcess.getDataObjects())
+                            .extracting(ValuedDataObject::getName, ValuedDataObject::getValue)
+                            .containsExactly(tuple("SubTest", "Testing"));
+                    assertThat(subProcess.getDataObjects().get(0).getItemSubjectRef().getStructureRef())
```

Comment:
No need for `isInstanceOfSatisfying` you can directly do `isEqualTo("xsd:string")`. If it is not a string then it won't be equal to a string as well

Reply:
Done.